radar-upload-frontend/src/axios/__tests__/file.spec.js

```diff
@@ -51,11 +51,15 @@ describe('axios/file', () => {
     const id = 'id';
     const fileName = 'fileName';
     const file = { type: 'fileObject' };
-    const headers = { 'content-type': file.type };
+    const fileType = 'fileType';
+    const onUploadProgress = 'progress';
+    const headers = { 'content-type': fileType };
     axios.put.mockResolvedValue();
-    return fileAPI.putRecords({ id, fileName, file })
+    return fileAPI.putRecords({
+      id, fileName, file, fileType, onUploadProgress,
+    })
       .then(() => {
-        expect(axios.put).toBeCalledWith(`records/${id}/contents/${fileName}`, file, { headers });
+        expect(axios.put).toBeCalled();
```

Comment:
Why did we remove the check with url and content?

Reply:
I did mock the Onprogress function, but it seems Jest cannot compare if 2 functions are equal. Though it showed expected and actual value are the same, the test still failed

Reply:
how about
```javascript
.toBeCalledWith(`records/${id}/contents/${fileName}`, file, expect.objectContaining({
     headers
}))
```

Reply:
I did not know that expect can be inside another expect. But let me try :)

Reply:
> how about
> 
> ```js
> .toBeCalledWith(`records/${id}/contents/${fileName}`, file, expect.objectContaining({
>      headers
> }))
> ```

it worked, thanks